adapters/algorix/algorixtest/exemplary/sample-banner-with-other-mediatype.json

```diff
@@ -0,0 +1,91 @@
+{
+  "mockBidRequest": {
+    "id": "test-request-id",
+    "imp": [
+      {
+        "id": "test-imp-id",
+        "banner": {
```

Comment:
Probably better as a multi-format request and bidding with the media type that is not the default assumption according to the original media type guessing logic.

Reply:
but PBS not support multi-format request.

Reply:
PBS has supported multi-format requests for years. That there is even a media type field in the PBS bid response is for multi-format requests.

Reply:
add multi-format requests.

Reply:
Can you move this to the /supplemental directory? This is not an example that publishers should follow and expect, though it does result in a bid.